honeycomb-core/src/twomap.rs

```diff
@@ -102,9 +100,9 @@ const TWO_MAP_BETA: usize = 3;
 /// let (v1, v2, v3): (VertexIdentifier, VertexIdentifier, VertexIdentifier) = (0, 1, 2);
 ///
 /// // place the vertices in space
-/// map.set_vertex(v1, [0.0, 0.0]);
-/// map.set_vertex(v2, [0.0, 10.0]);
-/// map.set_vertex(v3, [10.0, 0.0]);
+/// map.set_vertex(v1, [0.0, 0.0]).unwrap();
+/// map.set_vertex(v2, [0.0, 10.0]).unwrap();
+/// map.set_vertex(v3, [10.0, 0.0]).unwrap();
```

Comment:
Future work: proper error handling, with `?` for example.

Reply:
I tried this initially, but it seems the compiler isn't fond of having a returned value in doc tests. I'm open to suggestion on how to resolve this

Reply:
You can look at Métis-rs or coupe for examples.